### PR TITLE
[MIG][10.0-Issue175-T23298] ldap_password

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
Migrated ldap_password in v10.
@elicoidal @seb-elico @victormartinelicocorp Please review.
This PR refers to issue #175

### CODE REVIEW

File: ldap_password/__init__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © <2010-2015> <Augustin Cisterne-Kaas>
+# © 2017 Elico corp (www.elico-corp.com)
```
Comment (elicoidal) [inline ldap_password/__init__.py]: 2015-2017 (you need to keep the original date always)

File: ldap_password/__manifest__.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # -*- coding: utf-8 -*-
-# © 2015 Elico Corp (www.elico-corp.com)
+# © 2017 Elico corp (www.elico-corp.com)
```
Comment (elicoidal) [inline ldap_password/__manifest__.py]: 2015-2017 Elico Corp (https://www.elico-corp.com)

(same everywhere)

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details on my side.
Not sure about the demo data and the tests here

**Review by elicoidal — APPROVED** [review]

**Review by victor-ditay — APPROVED** [review]
To be tested

**Pull request merged**